RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitOOTECAL_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+from RecoParticleFlow.PFClusterProducer.particleFlowRecHitECAL_cfi import *
+
+particleFlowOOTRecHitECAL = particleFlowRecHitECAL.clone()
+particleFlowOOTRecHitECAL.producers[0].qualityTests[1].timingCleaning = cms.bool(False)
+particleFlowOOTRecHitECAL.producers[1].qualityTests[1].timingCleaning = cms.bool(False)
```

Comment:
this should have some protection that modifications are done correctly.
Simple access by index can easily fail or modify something unrelated.
The minimal change I can think of is to drop "cms.bool" so that this fails if it tries to modify a qualityTest without the timingCleaning parameter.
Additional checks will probably require consistency in inputs.